app/Repositories/Place/PlaceRepository.php

```diff
@@ -108,6 +110,40 @@ public function findCollectionByCriterias(CriteriaInterface ...$criterias): Coll
         return $result;
     }
 
+    public function setTranslations(Place $place, array $localizations): void
+    {
+        foreach ($localizations as $key => $value) {
+            $place->localization()->create([
+               'place_name'        => $value['name'],
+               'place_description' => $value['description'],
+               'place_id'          => $place->id,
+               'language_id'       => Language::where('code', $key)->first()->id
+            ]);
+        }
+    }
+
+    public function syncTags(Place $place, array $tags): void
+    {
+        $place->tags()->sync($tags);
+    }
+
+    public function syncFeatures(Place $place, array $features): void
+    {
+        $place->features()->sync($features);
+    }
+
+    public function setWorktime(Place $place, array $worktime): void
```

Comment:
// PlaceWorkTime models in action
new PlaceWorkTime([
           'place_id' => $place->id,
                'day_code' => $key,
                'start_time' => Carbon::parse($value['start'])->toDateTimeString(),
                'end_time' => Carbon::parse($value['end'])->toDateTimeString()
]);

setWorkTime(Place $place, Collection $worktimes)
{
  $place->worktime()->delete();
  $place->worktime()->saveMany(worktimes);
}